toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -475,3 +457,22 @@ func (r *RpmRepoCloner) clonePackage(baseArgs []string, enabledRepoOrder ...stri
 
 	return
 }
+
+func convertPackageVersionToTdnfArg(pkgVer *pkgjson.PackageVer) (tdnfArg string) {
+	tdnfArg = pkgVer.Name
+	// TDNF does not accept versioning information on implicit provides.
+	if pkgVer.IsImplicitPackage() {
+		if pkgVer.Condition != "" {
+			logger.Log.Warnf("Discarding version information on implicit package (%v)", pkgVer)
+		}
+		return
+	}
+
+	// Treat <= as =
+	// Treat > and >= as "latest"
+	if pkgVer.Condition == "=" || pkgVer.Condition == "<=" {
```

Comment:
Could we print an error message when this happens? Its something that we should make an effort to highlight since it can really mess stuff up. (Looking at you, node-js)

Reply:
(Doesn't need to actually error though)